packages/frontend-2/lib/workspaces/graphql/queries.ts

```diff
@@ -123,3 +123,11 @@ export const workspaceWizardRegionQuery = graphql(`
     }
   }
 `)
+
+export const workspaceGetIdBySlugQuery = graphql(`
```

Comment:
Some queries are not possible with workspaceBySlug, as nested properties in some cases need an ID and don't work with slug, so this call is done first to get the id

Reply:
Do you remember which hack this was, or maybe @andrewwallacespeckle knows? I vaguely remember it, but dont remember where we are doing this

Reply:
I don't know, I think Andrew will know more

Reply:
Worked around this by refactoring some of the child components to use workspaceBySlug